src/test/java/com/regnosys/testing/schemeimport/LatestSchemesImportTest.java

```diff
@@ -36,7 +36,7 @@ public void downloadLatestVersions() throws IOException, NoSuchAlgorithmExceptio
             fos.getChannel().transferFrom(rbc, 0, Long.MAX_VALUE);
             String checksum = getZipCheckSum(Paths.get(CODE_LIST_ZIP));
             assertThat("CodeList zip has been updated, run again with WRITE_LATEST_VERSION enabled then update expected checksum",
-                    checksum, equalTo("3c5ae6277ccee3b49792da10831cc010"));
+                    checksum, equalTo("f4e20644ca711aaa443c03980cb8b785"));
```

Comment:
what is this?

Reply:
There have been updates to the fpml schema. They have been updated as part of these changes to get the test to pass.